internal/store/logs.go

```diff
@@ -82,7 +85,26 @@ func (s *Store) Collect(groupID string, newLog *opsmngr.LogCollectionJob) (*opsm
 func (s *Store) DownloadLog(groupID, host, name string, out io.Writer, opts *atlas.DateRangetOptions) error {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		_, err := s.client.(*atlas.Client).Logs.Get(s.ctx, groupID, host, name, out, opts)
+		fileBaseName := strings.TrimSuffix(name, filepath.Ext(name))
+		result := s.clientv2.MonitoringAndLogsApi.GetHostLogs(s.ctx, groupID, host, fileBaseName)
+		if opts != nil {
+			if opts.StartDate != "" {
+				start, _ := strconv.ParseInt(opts.StartDate, 10, 64)
+				result = result.StartDate(start)
```

Comment:
if the parse operation above fails, what will be assigned here? nil or empty string? 

if opts.EndDate is nil, what will the code do?

Reply:
@swattyT I would recommend logging improvement jira to cover the date time for the atlas CLI and SDKs.
Both systems have different standards but are always the same so we can write familiar helper and do transformation directly at the Cobra flag level. That will let us leave complex `start, _ := strconv.ParseInt(opts.StartDate, 10, 64)` out of the future code reviews.